src/ESM.sol

```diff
@@ -61,21 +55,23 @@ contract ESM {
         require(z >= x);
     }
 
-    function fire() external note {
+    function fire() external {
         require(fired == 0,  "esm/already-fired");
         require(Sum >= min,  "esm/min-not-reached");
 
         end.cage();
 
         fired = 1;
+        emit Fire();
```

Comment:
Would it be worthwhile to emit `Sum` here?

Reply:
Hmmm in general should be just `min`, otherwise is a waste of MKR.
On the other hand is that a data value which actually adds value when having it?
Not opposed to add it if you consider is worth it.

Reply:
Tbh I'm not sure I see much value...what would a consumer of the event do with that info? Probably they just care about the fact that ES was triggered.